.circleci/config.yml

```diff
@@ -19,22 +19,6 @@ jobs:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
       - run: npm test
-  "node-latest":
```

Comment:
Why did this get simplified so much? Is it because you use `buildQuick` on CircleCI?

Reply:
The test was previously being executed against node 8 and node latest. What is being proposed here is to only test on node-8 to reduce chance of tests interfering with each other. 

I will investigate the possibility of running these serially instead.